spec/factories/board.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do 
+  factory :board do
+    name Faker::Hipster.word
+    description Faker::Hipster.sentence 
```

Comment:
I was having uniqueness issues here so I had to change it to `name { Faker::Hipster.unique.word }`